fuse/ipns/ipns_unix.go

```diff
@@ -238,7 +239,11 @@ func (s *Node) Attr() fuse.Attr {
 			size = 0
 		}
 		if size == 0 {
-			size = s.dagMod.Size()
+			dmsize, err := s.dagMod.Size()
+			if err != nil {
+				log.Error(err)
```

Comment:
It can fail if the underlying flush fails.. which is bad juju because that means the datastore is broken somehow. But theres not really anything we can do if it does fail, so I kinda just roll with it.